app/controllers/home_controller.rb

```diff
@@ -218,6 +218,29 @@ def tagged
     end
   end
 
+  def for_domain
```

Comment:
Nice way to present these. Do we want to drop the `domain:` operator or make it use the Domain model as well? I guess the latter? If I pulled stats on usage they'd be totally smashed by the aggressive scraping the other day, so we don't have clear signal that it's useful, but it seems worth supporting.

Reply:
I think it may be worth having a meta thread about search at some point to discuss this with the community as a whole. Leaving the current functionality in place doesn't make the situation any worse than it is right now.

Reply:
Yeah. Can you update `Search#with_stories_in_domain` to use the new model?